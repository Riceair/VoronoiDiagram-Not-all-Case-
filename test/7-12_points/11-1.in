11
359 140
164 319
576 417
131 162
228 487
448 261
219 165
447 561
31 609
218 381
541 160