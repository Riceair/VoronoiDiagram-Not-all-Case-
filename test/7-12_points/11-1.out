P 31 609
P 131 162
P 164 319
P 218 381
P 219 165
P 228 487
P 359 140
P 447 561
P 448 261
P 541 160
P 576 417
E -281529 -62614.5 -221 317.94
E -15461 -88047.5 309.88 269.46
E -221 317.94 69.52 451.18
E -221 317.94 172.53 235.22
E 69.52 451.18 78.95 447.59
E 69.52 451.18 258.83 756.86
E 78.95 447.59 279.09 273.28
E 78.95 447.59 372.69 419.89
E 172.53 235.22 279.09 273.28
E 172.53 235.22 2065 -55276.5
E 258.83 756.86 372.69 419.89
E 258.83 756.86 372.69 419.89
E 258.83 756.86 30479 262665
E 279.09 273.28 308.22 273.41
E 308.22 273.41 309.88 269.46
E 308.22 273.41 379.88 410.75
E 309.88 269.46 448.11 167.78
E 372.69 419.89 379.88 410.75
E 379.88 410.75 424.34 410.91
E 424.34 410.91 576.54 286.04
E 424.34 410.91 91231.5 81759
E 448.11 167.78 576.54 286.04
E 448.11 167.78 13050 -114510
E 576.54 286.04 162486.5 -21763.96
